compiler/src/iree/compiler/Codegen/Common/Passes.td

```diff
@@ -490,6 +490,11 @@ def RemoveSingleIterationLoopPass :
   let summary = "Remove distributed loop with single iteration.";
 }
 
+def StripConfigInfoPass :
+    InterfacePass<"iree-codegen-strip-config-info", "mlir::FunctionOpInterface">{
+   let summary = "Remove all the the lowering configuration and translation info.";
```

Comment:
I'd call it 'StripCompilationInfo`. The reason is that the `#iree_codegen.compilation_info` attribute contains both lowering config and translation info, so it that it encompasses all 3 attributes

Reply:
Or `StripConfiguration`

Reply:
good call